lib/google_drive.rb

```diff
@@ -86,70 +84,50 @@ def self.login_with_oauth(client_or_access_token, proxy = nil)
     # +path+ defaults to ENV["HOME"] + "/.ruby_google_drive.token".
     #
     # You can specify your own OAuth +client_id+ and +client_secret+. Otherwise the default one is used.
-    def self.saved_session(path = nil, proxy = nil, client_id = nil, client_secret = nil)
+    def self.saved_session(path = './.config.json', proxy = nil, client_id = nil, client_secret = nil)
+      config = Config.new(path)
+
+      if client_id.nil? || client_secret.nil?
+        config.call
```

Comment:
Do you have any reason to alias "load_config_file" as "call"? Can you just call config.load_config_file here directly? "load_config_file" sounds more intuitive than "call".